### DISCUSSION TITLE
Analog dial going over digital dial - affecting reading

### DISCUSSION THREAD

**Discussion by amiler** [jomjol/AI-on-the-edge-device#1989] [Q&A]
I have Neptune H65N watermeter where analog dial (needle) is going over the digital (mechanical) dial (attached image). When analog red dial is over the mechanical dial, the reading is affected since it covers part of the number that is been recognized. The analog dial does one full rotation until the last digit changes. Reading is in m3 (white numbers) while black number on the right is in 1/10 of m3 and analog dial is 1/100 of m3. 
I am using network that should detect partial rotation of the mechanical dial (100 classes rather than 10) but, in many cases, recognition of partial turn of the dial is not properly recognized.

Any suggestion how to improve reading accuracy and avoid this problem of analog dial stopping over the mechanical dial affecting number recognition as well as incorrect partial turn recognition?
Attached image shows how my watermeter looks
![neptune_watermeter](https://user-images.githubusercontent.com/75551075/217026147-57853f57-17fe-42e4-b529-1dccd8cd1c08.JPG)

**Comment by beyondunreal79** [comment]
Similar issue here.

**Comment by caco3** [comment]
relates to https://github.com/jomjol/AI-on-the-edge-device/discussions/2682
